code/game/mecha/equipment/weapons/ranged/laser_weapons.dm

```diff
@@ -50,34 +52,42 @@
 	name = "\improper Multi-laser Cannon"
 	desc = "Four solaris laser cannons welded together to achieve a higher firerate."
 	icon_state = "mecha_auto_laser"
-	energy_drain = 180
+	matter = list(MATERIAL_STEEL = 10, MATERIAL_PLASTEEL = 5, MATERIAL_SILVER  =1)
+	energy_drain = 500
 	projectile = /obj/item/projectile/beam/heavylaser
 	projectiles = 8
 	projectiles_per_shot = 2
-	fire_cooldown = 1
+	fire_cooldown = 5 //so we dont fire all our beams at the exact same instant. Remember! We track once a target is clicked!
 	fire_sound = 'sound/weapons/lasercannonfire.ogg'
 
 /obj/item/mecha_parts/mecha_equipment/ranged_weapon/energy/ion
-	equip_cooldown = 40
+	equip_cooldown = 40 //this is fine. This is a very dangerous tool.
 	name = "mkIV ion heavy cannon"
 	icon_state = "mecha_ion"
-	energy_drain = 120
-	projectile = /obj/item/projectile/ion
-	fire_sound = 'sound/weapons/Laser.ogg'
-	price_tag = 300
+	/obj/item/mecha_parts/mecha_equipment/ranged_weapon/energy/laser/heavy
```

Comment:
```suggestion
/obj/item/mecha_parts/mecha_equipment/ranged_weapon/energy/laser/heavy
```

Reply:
The weapon was already defined above the multi-laser

Reply:
oh huh, must've accidentally moved something about